library/Kubernetes/Web/ViewModeSwitcher.php

```diff
@@ -0,0 +1,245 @@
+<?php
+
+/* Icinga for Kubernetes Web | (c) 2025 Icinga GmbH | AGPLv3 */
+
+namespace Icinga\Module\Kubernetes\Web;
+
+use ipl\Html\Attributes;
+use ipl\Html\Form;
+use ipl\Html\FormElement\HiddenElement;
+use ipl\Html\FormElement\InputElement;
+use ipl\Html\HtmlElement;
+use ipl\Web\Common\FormUid;
+use ipl\Web\Widget\IcingaIcon;
+
+class ViewModeSwitcher extends Form
```

Comment:
* `use Translation`
* Use features up to PHP 8.2., e.g., property types, view mode enum, static return type.